src/mpol/fourier.py

```diff
@@ -61,6 +63,9 @@ def forward(self, cube):
         # See MPoL documentation and/or TMS Eqn A8.18 for more information.
         self.vis = self.coords.cell_size**2 * torch.fft.fftn(cube, dim=(1, 2))
 
+        self.register_buffer("vis", 
```

Comment:
I don't think this extra line needs to be here.

Reply:
Removed.